vagrant/provisioning/roles/alfresco7/tasks/apply-amp.yml

```diff
@@ -14,4 +14,23 @@
       become_user: alfresco7
       command: java -jar "{{ root_folder }}/app/alfresco7/bin/alfresco-mmt.jar" install {{ amp.amp_path }} "{{ root_folder }}/app/alfresco7/webapps/{{ amp.target_war }}"
       when: 'amp.title not in installed_amps.stdout'
+  when: alfresco_ee_version is version('7.3.0.1', '<')
```

Comment:
@david-oc-miller @dzdimov Do we need the option for older alfresco versions? 7.3 is working with java-11, and if we are going with alfresco7.3 from now on, i can just leave one set of tasks here, only for java-11, and delete the ones with java8

Reply:
I don't know; we would have to try it and see.  When we want to run the installer to fix some issue with existing, older Alfresco installations, but we don't want to upgrade to the newer Alfresco, then it makes sense to continue supporting older Alfresco versions.